examples/burial-form/data.ts

```diff
@@ -0,0 +1,13 @@
+export const RadioLabels = {
```

Comment:
Would it be alright to put this in something like a `constants.ts` file? I feel like keeping it in data.ts is a little misleading since it's more of a static object.